src/components/SlateEditor/plugins/concept/SearchConceptForm.tsx

```diff
@@ -25,13 +27,12 @@ interface Props {
   subjects: Node[];
   searchObject: SearchParams;
   locale: string;
-  userData?: IUserData | undefined;
-  disableSavedSearch?: boolean;
+  userData: IUserData | undefined;
 }
 
-const SearchConceptForm = ({ search, searchObject, subjects, userData, disableSavedSearch }: Props) => {
+const SearchConceptForm = ({ search: doSearch, searchObject: search, subjects, userData }: Props) => {
```

Comment:
Synes denne denne renamingen gjør det vanskeligere å følge attributter inn i objektet. Det som heter search inn i SearchConceptForm er ikke det samme som heter search inne i komponenten. Med mindre det er en veldig god grunn te at disse navnene må flippes her, synes jeg vi bør etterstrebe lik navngivning.

Reply:
Katrine gjorde jo dette, men det unnslapp denne pr for den har ligge ganske lenge. Kan oppdatere.